server/src/main.rs

```diff
@@ -105,9 +105,10 @@ impl WipeMode {
             Some(WipeMode::LocalLoglet) => {
                 restate_fs_util::remove_dir_all_if_exists(config.bifrost.local.data_dir()).await?
             }
-            Some(WipeMode::MetadataServer) => {
-                restate_fs_util::remove_dir_all_if_exists(config.metadata_server.data_dir()).await?
-            }
+            Some(WipeMode::MetadataServer) => restate_fs_util::remove_dir_all_if_exists(
+                restate_metadata_server::local::data_dir(),
+            )
+            .await?,
```

Comment:
Interesting, we don't wipe the replicated loglet. Unsure if we should or if we should remove support for this dangerous option.

Reply:
Right now the wipe option is hidden. Maybe it's about time to remove it completely.